ui/raidboss/data/04-sb/ultimate/ultima_weapon_ultimate.js

```diff
@@ -15,40 +15,44 @@
         fr: 'Bougez',
         ja: 'フェザーレイン',
         ko: '이동',
+        cn: '躲羽毛',
       },
     },
   ],
   triggers: [
     // Phases
     {
+      regexKo: /:알테마 웨폰:2D4D:/,
```

Comment:
Sorry, but can you put `regex:` on its own first? (and maybe `regexKo` last, because of alphabetical sorting?)

Reply:
Yeah that's because of no `ko` entry in the `check_translation.py` script, I've add it to the script.